lib/meta_tags/text_normalizer.rb

```diff
@@ -94,7 +95,16 @@ def self.safe_join(array, sep = $,)
     # space characters squashed into a single space.
     #
     def self.cleanup_string(string)
-      strip_tags(string).gsub(/\s+/, ' ').strip.html_safe
+      html_escape(strip_tags(string)).gsub(/\s+/, ' ').strip.html_safe
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.